app/models/metric/ci_mixin/capture.rb

```diff
@@ -8,8 +8,10 @@ def perf_capture_object
   def queue_name_for_metrics_collection
     ems = if self.kind_of?(ExtManagementSystem)
             self
-          elsif self.respond_to?(:ext_management_system)
+          elsif respond_to?(:ext_management_system) && ext_management_system.present?
             ext_management_system
+          elsif respond_to?(:old_ext_management_system) && old_ext_management_system.present?
+            old_ext_management_system
```

Comment:
This concerns me as queue_name_for_metrics_collection targets which worker will run the collection.  If old_ext_management_system points to a no longer enabled EMS, what will happen?  Will we just put stuff on the queue that will never get executed?